packages/gguf/src/types.ts

```diff
@@ -66,39 +57,18 @@ interface General {
 	"general.quantization_version": number;
 }
 
-type Attention<TArchitecture extends Architecture> =
-	| { [K in `${TArchitecture}.attention.head_count`]: number }
-	| { [K in `${TArchitecture}.attention.head_count_kv`]: number }
-	| { [K in `${TArchitecture}.attention.layer_norm_epsilon`]: number }
-	| { [K in `${TArchitecture}.attention.layer_norm_rms_epsilon`]: number }
-	| { [K in `${TArchitecture}.attention.alibi_bias_max`]: number }
-	| { [K in `${TArchitecture}.attention.clip_kqv`]: number }
-	| { [K in `${TArchitecture}.attention.use_norm`]: number };
-
-type Rope<TArchitecture extends Architecture> =
-	| { [K in `${TArchitecture}.rope.dimension_count`]: number }
-	| { [K in `${TArchitecture}.rope.freq_base`]: number }
-	| { [K in `${TArchitecture}.rope.scale`]: number }
-	| { [K in `${TArchitecture}.rope.scale_linear`]: number };
-
-type ModelBase<
+export type ModelBase<
 	TArchitecture extends
 		| Architecture
 		| `encoder.${Extract<Architecture, "whisper">}`
 		| `decoder.${Extract<Architecture, "whisper">}`,
```

Comment:
@mishig25 is this on purpose? 

`TArchitecture` will always extend `"encoder.whisper" | "decoder.whisper"`

Reply:
I couldn't find any whisper gguf model on the internet, so maybe we should temporary remove support for whisper and we'll see later?

More info: there're some [ggml models uploaded by ggerganov](https://huggingface.co/ggerganov/whisper.cpp/tree/main), but they uses ggml format, not gguf.

Reply:
the whisper spec of gguf is here https://github.com/ggerganov/ggml/blob/master/docs/gguf.md#whisper

Reply:
> @mishig25 is this on purpose?

yes